Makefile

```diff
@@ -135,3 +135,16 @@ build: ## Create GO vendor directories with all dependencies
 build-release-template: ## Build KFP Tekton release deployment templates
 	@mkdir -p install/$(KFP_TEKTON_RELEASE)
 	@kustomize build manifests/kustomize/env/kfp-template -o install/$(KFP_TEKTON_RELEASE)/kfp-tekton.yaml
+
+.PHONY: build-backend
+build-backend: apiserver agent workflow ## Verify apiserver, agent, and workflow build
+	@echo "$@: OK"
+
+apiserver: ## Build apiserver
```

Comment:
can we add the `build-` prefix to these make commands? so it's more clear if someone wants to run it independently.

Reply:
Done!